### PR TITLE
Renaming work

### PR DESCRIPTION
**Pull request by whitej6**

### CODE REVIEW

**Comment by jdrew82** [comment]
This can't be the only thing changing is it?

File: README.md [hunk 1]
```diff
@@ -63,15 +63,15 @@ Sign up [here](https://slack.networktocode.com/)
 
 ## Acknowledgements
 
-This project includes code originally written in separate plugins, which have been merged into this project:
+This project includes code originally written in separate apps, which have been merged into this project:
 
-- [nautobot-plugin-chatops-aci](https://github.com/nautobot/nautobot-plugin-chatops-aci):
```
Comment (smk4664) [inline README.md:68]: Since these are all archived, I don't think we need to change the name of them.

File: README.md [hunk 2]
```diff
@@ -63,15 +63,15 @@ Sign up [here](https://slack.networktocode.com/)
...
+- [nautobot-plugin-chatops-aci](https://github.com/nautobot/nautobot-app-chatops-aci):
     Thanks
     [@mamullen13316](https://github.com/mamullen13316),
     [@smk4664](https://github.com/smk4664),
     [@ubajze](https://github.com/ubajze),
     [@whitej6](https://github.com/whitej6),
-- [nautobot-plugin-chatops-ansible](https://github.com/nautobot/nautobot-plugin-chatops-ansible):
+- [nautobot-plugin-chatops-ansible](https://github.com/nautobot/nautobot-app-chatops-ansible):
```
Comment (smk4664) [inline README.md:74]: See above

File: README.md [hunk 3]
```diff
@@ -81,20 +81,20 @@ This project includes code originally written in separate plugins, which have be
     [@smk4664](https://github.com/smk4664),
     [@ubajze](https://github.com/ubajze),
     [@whitej6](https://github.com/whitej6),
-- [nautobot-plugin-chatops-arista-cloudvision](https://github.com/nautobot/nautobot-plugin-chatops-arista-cloudvision):
+- [nautobot-plugin-chatops-arista-cloudvision](https://github.com/nautobot/nautobot-app-chatops-arista-cloudvision):
```
Comment (smk4664) [inline README.md:84]: See above

File: README.md [hunk 4]
```diff
@@ -81,20 +81,20 @@ This project includes code originally written in separate plugins, which have be
...
     Thanks
     [@qduk](https://github.com/qduk),
     [@ubajze](https://github.com/ubajze),
     [@whitej6](https://github.com/whitej6),
-- [nautobot-plugin-chatops-grafana](https://github.com/nautobot/nautobot-plugin-chatops-grafana):
+- [nautobot-plugin-chatops-grafana](https://github.com/nautobot/nautobot-app-chatops-grafana):
```
Comment (smk4664) [inline README.md:89]: See above

File: README.md [hunk 5]
```diff
@@ -81,20 +81,20 @@ This project includes code originally written in separate plugins, which have be
...
     Thanks
     [@jedelman8](https://github.com/jedelman8),
     [@josh-silvas](https://github.com/josh-silvas),
     [@nniehoff](https://github.com/nniehoff),
     [@tim-fiola](https://github.com/tim-fiola),
     [@ubajze](https://github.com/ubajze),
     [@whitej6](https://github.com/whitej6),
-- [nautobot-plugin-chatops-ipfabric](https://github.com/nautobot/nautobot-plugin-chatops-ipfabric):
+- [nautobot-plugin-chatops-ipfabric](https://github.com/nautobot/nautobot-app-chatops-ipfabric):
```
Comment (smk4664) [inline README.md:97]: See above

File: README.md [hunk 6]
```diff
@@ -106,15 +106,15 @@ This project includes code originally written in separate plugins, which have be
     [@smk4664](https://github.com/smk4664),
     [@ubajze](https://github.com/ubajze),
     [@whitej6](https://github.com/whitej6),
-- [nautobot-plugin-chatops-meraki](https://github.com/nautobot/nautobot-plugin-chatops-meraki):
+- [nautobot-plugin-chatops-meraki](https://github.com/nautobot/nautobot-app-chatops-meraki):
```
Comment (smk4664) [inline README.md:109]: See above

File: README.md [hunk 7]
```diff
@@ -106,15 +106,15 @@ This project includes code originally written in separate plugins, which have be
...
     Thanks
     [@jedelman8](https://github.com/jedelman8),
     [@jeffkala](https://github.com/jeffkala),
     [@qduk](https://github.com/qduk),
     [@tim-fiola](https://github.com/tim-fiola),
     [@ubajze](https://github.com/ubajze),
     [@whitej6](https://github.com/whitej6),
-- [nautobot-plugin-chatops-panorama](https://github.com/nautobot/nautobot-plugin-chatops-panorama):
+- [nautobot-plugin-chatops-panorama](https://github.com/nautobot/nautobot-app-chatops-panorama):
```
Comment (smk4664) [inline README.md:117]: See above

File: docs/models/chatopsaccountlink.md [hunk 8]
```diff
@@ -3,7 +3,7 @@
 <!--account-link-->
 +++3.0.0
 
-Nautobot ChatOps now uses the built-in Nautobot permissions for Nautobot Objects (Devices, Locations, Racks, etc.). Each user will need to link their Nautobot Account with their Chat Platform User Account. Login to Nautobot then access the Link ChatOps Account within the Plugins menu. Here you can provide your email address and select the ChatOps Platform you are using, then click the Look up User ID from Email to get your Chat User ID.
+Nautobot ChatOps now uses the built-in Nautobot permissions for Nautobot Objects (Devices, Locations, Racks, etc.). Each user will need to link their Nautobot Account with their Chat Platform User Account. Login to Nautobot then access the Link ChatOps Account within the Apps menu. Here you can provide your email address and select the ChatOps Platform you are using, then click the Look up User ID from Email to get your Chat User ID.
```
Comment (smk4664) [inline docs/models/chatopsaccountlink.md:6]: Is the Plugins menu in Nautobot being renamed? If not, I think this change should be reverted.

File: docs/models/commandtoken.md [hunk 9]
```diff
@@ -7,8 +7,8 @@ Some platforms this `signing_secret` is valid for all commands, other platforms,
 create a separate `token` for every slash command.  Keeping the records for Mattermost tokens in the
 `.creds.env` file would not be sustainable.  
 
-To solve this issue, the plugin has the option to store Command Tokens to the Nautobot Database.
-In Nautobot, open Nautobot and go to the Plugins and select Command Tokens. Below is an example to
+To solve this issue, the app has the option to store Command Tokens to the Nautobot Database.
+In Nautobot, open Nautobot and go to the Apps and select Command Tokens. Below is an example to
```
Comment (smk4664) [inline docs/models/commandtoken.md:11]: Same as other menu comment, this is referring to the "Plugins" dropdown on Nautobot.

File: docs/user/app_getting_started.md [hunk 10]
```diff
@@ -10,7 +10,7 @@ To install the App, please follow the instructions detailed in the [Installation
 
 +++3.0.0
 
-Nautobot ChatOps now uses the built-in Nautobot permissions for Nautobot Objects (Devices, Locations, Racks, etc.). Each user will need to link their Nautobot Account with their Chat Platform User Account. Login to Nautobot then access the Link ChatOps Account within the Plugins menu. Here you can provide your email address and select the ChatOps Platform you are using, then click the Look up User ID from Email to get your Chat User ID.
+Nautobot ChatOps now uses the built-in Nautobot permissions for Nautobot Objects (Devices, Locations, Racks, etc.). Each user will need to link their Nautobot Account with their Chat Platform User Account. Login to Nautobot then access the Link ChatOps Account within the Apps menu. Here you can provide your email address and select the ChatOps Platform you are using, then click the Look up User ID from Email to get your Chat User ID.
```
Comment (smk4664) [inline docs/user/app_getting_started.md:13]: Same as other menu comment, this is referring to the "Plugins" dropdown on Nautobot.

**Review by smk4664 — CHANGES_REQUESTED** [review]
Left a few comments on changes.

File: README.md (hunk 7 above)
Comment (whitej6) [reply README.md:117]: Oof I did a replace on `nautobot/nautobot-plugin-chatops` so I picked up the old repos. Good catch.

**Comment by whitej6** [comment]
> This can't be the only thing changing is it?

@jdrew82 yep, this is mostly cosmetic.

**Review by smk4664 — APPROVED** [review]
👍

**Pull request merged**